sponge/pom.xml

```diff
@@ -42,20 +42,28 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.9</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
         </plugins>
     </build>
 
     <dependencies>
         <dependency>
             <groupId>com.vexsoftware</groupId>
             <artifactId>nuvotifier-common</artifactId>
-            <version>${project.parent.version}</version>
+            <version>2.3.5</version>
```

Comment:
Why was this changed?

Reply:
Accidental, was trying to figure out how the project worked.